### PR TITLE
Update style issues found by newer eslint

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

File: tools/run-all-js-tests.py [hunk 1]
```diff
@@ -26,7 +26,8 @@ def main():
   lsfiles = subprocess.check_output(['git', 'ls-files']).split('\n')
   lsfiles.sort(key = len) # Sort by shortest first to make sure we deal with parents first
   for f in lsfiles:
-    path, name = os.path.split(f)
+    path, name = os.path.split(os.path.abspath(f))
```
Comment (arv) [inline tools/run-all-js-tests.py]: Now this correctly works from subdirectories too:

```
$ cd samples/js
$ python ../../tools/run-all-js-tests.py
```

File: js/noms/src/type-cache.js [hunk 2]
```diff
@@ -92,8 +92,8 @@ export default class TypeCache {
     return this.makeStructTypeQuickly(name, fieldNames, fieldTypes);
   }
 
-  makeStructTypeQuickly(name: string, fieldNames: Array<string>, fieldTypes: Array<Type<any>>):
-      Type<StructDesc> {
+  makeStructTypeQuickly(name: string, fieldNames: Array<string>, fieldTypes: Array<Type<any>>)
+    : Type<StructDesc> {
```
Comment (kalman) [inline js/noms/src/type-cache.js]: This looks different to the others.

File: js/noms/src/sequence.js [hunk 3]
```diff
@@ -252,7 +252,7 @@ export class SequenceIterator<T, S: Sequence<any>> extends AsyncIterator<T> {
   // Wraps |_advance|.then() with the guarantee that |_advance| hasn't changed since running .then()
   // and the callback being run.
   _safeAdvance(fn: (success: boolean) => AsyncIteratorResult<T> | Promise<AsyncIteratorResult<T>>)
-              :Promise<AsyncIteratorResult<T>> {
+              : Promise<AsyncIteratorResult<T>> {
```
Comment (kalman) [inline js/noms/src/sequence.js]: This looks different to the others

**Comment by kalman** [comment]
(Looks like there are a couple of jenkins failures though)

**Comment by arv** [comment]
Fixed.

PTAL

**Comment by kalman** [comment]
lgtm but I did already approve :-)

**Comment by arv** [comment]
So you did... Still not used to the new ui...

**Pull request merged**